### ISSUE TITLE
Suggestion to add blind accessibility features

### ISSUE THREAD

**Issue by Computerfan23** [TakaRikka/dusk#1895]
Hello there!

I am a blind accessibility player who loves zelda games. I have heard about the dusk project and i want to ask if it is possible to make a suggestion to add blind accessibility features. Since i am literally blind and cannot play gmaes too well without these features.

First, i would require some form to read the game text which include dialogue, menu options and other text through some use of text to speech function.
Second, i require some way of navigation like if you could make some form of pathfinder which means to guide the character with path to certain things, like you have some audio beacon.
Third, i would require sounds for enemies to know where exactly they are located, or a way to find them easily.
Fourth, i would require sounds for doors and exits from buildings and other areas like map transitions.
Fifth, i would need sounds for item pickups like hearts, rupees, ammo etc.
Sixth, i would require some form of sound for npc, like you could make a filter for what character i want to hear.
Seventh, for important things like golden bugs, poe souls, tears of light i would require some form of sound.
Eight, i would require some sound for ledges and platforms. Basically make i so i can determine where platforms, ledges and gaps are.

So all in all, i don't know how possible it would be to make these features, but i wanted to make this issue.

**Comment by PJB3005** [comment]
While I'm personally open to making the game as accessible as possible, I'm sure you understand that what you're asking for here is a *lot*. I know many games have features to improve accessibility for people with visual impairments, but I'm not aware of any 3D AAA games of this scale that are designed to be completeable by people who are fully blind. Do you have any examples of any such games we could use to get a better reference of what you're imagining?

**Comment by Computerfan23** [comment]
Yes, i do have some examples for you regarding games that blind players are able to complete. You could check out The Last of Us series, both of the games in the sereis have full accessibility features for blind and low vision players. I also know God of War: Ragnarok also has blind accessibility. I fully understand i may have suggested a lot of things in this issue and i understand that.
